dist/tools/pyterm/pyterm

```diff
@@ -286,7 +286,7 @@ class SerCmd(cmd.Cmd):
 
         # start serial->console thread
         receiver_thread = threading.Thread(target=self.reader)
-        receiver_thread.setDaemon(1)
+        receiver_thread.daemon = True
```

Comment:
Alternatively, as far as I understand [the doc](https://docs.python.org/3/library/threading.html#threading.Thread) you could also set `daemon` in the constructor. That one is confirmed to be supported since Python 3.3 in the doc (in case anyone wants to use `pyterm` e.g. on the A8 nodes, where IIRC there is still Python 3.5):

```suggestion
        receiver_thread = threading.Thread(target=self.reader, daemon=True)
```

Reply:
That's a good suggestion. I've pushed a fixup commit